### ISSUE TITLE
Learn/update not usable since migration to argparse

### ISSUE THREAD

**Issue by lytenyn** [afewmail/afew#136]
As far as I can tell, the migration to argparse somehow made the options to -l and -u vanish.
Symptom:

    % afew --learn spam tag:spam

    Traceback (most recent call last):
      File ".local/bin/afew", line 11, in <module>
        load_entry_point('afew==1.0.0', 'console_scripts', 'afew')()
      File "/home/jan/.local/lib64/python3.4/site-packages/afew-1.0.0-py3.4.egg/afew/commands.py", line 205, in main
        inner_main(args, database, query_string)
      File "/home/jan/.local/lib64/python3.4/site-packages/afew-1.0.0-py3.4.egg/afew/main.py", line 49, in main
        database.mail_bodies_matching(query_string)
      File "/home/jan/.local/lib64/python3.4/site-packages/afew-1.0.0-py3.4.egg/afew/DBACL.py", line 78, in learn
        process = self._call_dbacl(['-l', self.get_category_path(category)])
      File "/home/jan/.local/lib64/python3.4/site-packages/afew-1.0.0-py3.4.egg/afew/DBACL.py", line 75, in get_category_path
        return os.path.join(self.database_directory, category.replace('/', '_'))
    AttributeError: 'bool' object has no attribute 'replace'

This is with the latest git revision.

**Comment by GuillaumeSeren** [comment]
Hey,
I confirm the bug, this should be fixed, PR are welcome but I will look at it asap
```
 afew --learn spam tag:spam
Traceback (most recent call last):
  File "/home/gseren/.local/bin/afew", line 11, in <module>
    load_entry_point('afew==1.0.0', 'console_scripts', 'afew')()
  File "/home/gseren/.local/lib64/python3.4/site-packages/afew-1.0.0-py3.4.egg/afew/commands.py", line 205, in main
    inner_main(args, database, query_string)
  File "/home/gseren/.local/lib64/python3.4/site-packages/afew-1.0.0-py3.4.egg/afew/main.py", line 49, in main
    database.mail_bodies_matching(query_string)
  File "/home/gseren/.local/lib64/python3.4/site-packages/afew-1.0.0-py3.4.egg/afew/DBACL.py", line 78, in learn
    process = self._call_dbacl(['-l', self.get_category_path(category)])
  File "/home/gseren/.local/lib64/python3.4/site-packages/afew-1.0.0-py3.4.egg/afew/DBACL.py", line 75, in get_category_path
    return os.path.join(self.database_directory, category.replace('/', '_'))
AttributeError: 'bool' object has no attribute 'replace'
```

**Comment by GuillaumeSeren** [comment]
This is about those line in the afew/DBACL.py (arround line 74)
```py
    def get_category_path(self, category):
        return os.path.join(self.database_directory, category.replace('/', '_'))
```

**Comment by opensamba** [comment]
Hi,

I pushed a PR trying to fix the issue #138

I double checked the prev code and seems after the argparse migration the learn option was not configured correctly.

I tested and seems it's working fine for me.
